.gitmodules

```diff
@@ -25,3 +25,6 @@
 [submodule "lib/libpng/libpng"]
 	path = lib/libpng/libpng
 	url = https://github.com/glennrp/libpng.git
+[submodule "lib/libjpeg/libjpeg-turbo"]
+	path = lib/libjpeg/libjpeg-turbo
+	url = https://github.com/libjpeg-turbo/libjpeg-turbo.git
```

Comment:
I just realized that the commits which add the packages have no prefix in their names. I'd expect them to have a prefix like "libjpeg:" and "sdl:" / "sdl_image:" / ...

Also see earlier commits:
- https://github.com/XboxDev/nxdk/commit/632fe50b4017c523d9598a7d2782748e7a4ee227
- https://github.com/XboxDev/nxdk/commit/c40c34bc61b75ed2f5da01a9296be1f1326d6c5e

Reply:
I added a prefix to the two commits now.